Sources/Kitura/RouterRequest.swift

```diff
@@ -32,7 +32,7 @@ public class RouterRequest: SocketReader {
     /// The hostname of the request
     ///
     public var hostname: String {
-        if  let host = headers["host"] {
+        if  let host = headers.get("host")?.first {
```

Comment:
I can add a subscript shorthand so headers["host"] is equivalent to headers.get("host"). It doesn't mean that headers is a full fledged dictionary though.